utils/functions/dbChangesLogger.js

```diff
@@ -0,0 +1,49 @@
+import logger from "@config/logger";
```

Comment:
Please update your branch, because the `utils` directory has been removed. I suggest moving this re-usable function to the `models` directory as it will only be used in there (maybe to `model/index.js` for now)

Reply:
So, I moved the function to a new directory at `models/middlewares` instead, I thought it wouldn't be nice to let the `models/index.js` deviate from its simple functionality of group exporting the models.

> ...the `models` directory as it will only be used in there...

This way we are making sure the `dbChangesLoggerMiddleware` is not in any way made available to other file during `import` from `models` since it will only be used by `models`

```js
import { dbChangesLoggerMiddleware } from "@models/index";
```

If it will be imported for any reason, then it must be intentional

```js
import dbChangesLoggerMiddleware from "@models/middlewares/dbChangesLogger";
```

Reply:
Ok thank you 👍